### ISSUE TITLE
Version 0.10.0 does not update to newer version

### ISSUE THREAD

**Issue by flashman2** [jhen0409/react-native-debugger#408]
Previously, update of React Native Debugger worked just fine!  But currently I have 0.10.0 and when I press "Check for updates" I see a message "A newer version is available 0.10.1"
When I press "Download" button nothing happens and new version is not installed. I guess, there is a problem with update functionality in 0.10.0

**Comment by FrankyHo** [comment]
> Previously, update of React Native Debugger worked just fine! But currently I have 0.10.0 and when I press "Check for updates" I see a message "A newer version is available 0.10.1"
> When I press "Download" button nothing happens and new version is not installed. I guess, there is a problem with update functionality in 0.10.0

@flashman2 try "brew cask reinstall react-native-debugger"

**Comment by LaVielle** [comment]
@FrankyHo Thanks that fixed it for me!
Got a little unnerved when I saw the evil popup come again... but this time I got another one after a few seconds prompting to restart. Pffew! 😅

**Comment by octopicorn** [comment]
`brew cask` method does not work:

```
brew update && brew cask reinstall react-native-debugger
Already up-to-date.
==> Satisfying dependencies
==> Downloading https://github.com/jhen0409/react-native-debugger/releases/download/v0.10.1/rn-debugger-macos-x64.zip
Already downloaded: /Users/foo/Library/Caches/Homebrew/downloads/0efc6c9999b6e9d52efe4cccc721a951b5a053e1af73ae715532b8e4d8b8e673--rn-debugger-macos-x64.zip
==> Verifying SHA-256 checksum for Cask 'react-native-debugger'.
==> Installing Cask react-native-debugger
==> Purging files for version 0.10.1 of Cask react-native-debugger
Error: Permission denied @ dir_s_mkdir - /usr/local/Caskroom/react-native-debugger
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:254:in `mkdir'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:254:in `fu_mkdir'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:228:in `block (2 levels) in mkdir_p'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:226:in `reverse_each'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:226:in `block in mkdir_p'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:211:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/fileutils.rb:211:in `mkdir_p'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/pathname.rb:576:in `mkpath'
/usr/local/Homebrew/Library/Homebrew/unpack_strategy.rb:122:in `extract'
/usr/local/Homebrew/Library/Homebrew/unpack_strategy.rb:143:in `block in extract_nestedly'
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/tmpdir.rb:89:in `mktmpdir'
/usr/local/Homebrew/Library/Homebrew/unpack_strategy.rb:127:in `extract_nestedly'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:189:in `extract_primary_container'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:77:in `stage'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:99:in `install'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:126:in `reinstall'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/reinstall.rb:13:in `block in run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/reinstall.rb:7:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/reinstall.rb:7:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/abstract_command.rb:36:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:92:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:158:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:123:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:9:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:102:in `<main>'
```

**Comment by flashman2** [comment]
I ended up with switch to version 0.9.14 as it is more stable in work.

**Comment by brendanmc6** [comment]
I can confirm that me and several teammates have this same issue. `brew cask reinstall react-native-debugger` seems to have fixed it 👍

**Comment by jhen0409** [comment]
The auto update issue was fixed by [`v0.10.2`](https://github.com/jhen0409/react-native-debugger/releases/tag/v0.10.2) with https://github.com/jhen0409/react-native-debugger/commit/d6a46058ab1d780ee7e207fd4be61f0ecb0ca8b5. Please update it manually.
